### DISCUSSION TITLE
Respect Terminal Theme

### DISCUSSION THREAD

**Discussion by azzamsa** [Julien-cpsn/ATAC#100] [Q&A] [answered]
Hi, 👋🏽 

Can ATAC respect/use terminal color scheme?
Some buttons are unreadable.

![image](https://github.com/user-attachments/assets/418d852e-224b-49c5-b17c-97c80621dc90)

Thanks 🥞

**Comment by ccoVeille** [comment]
This could be used

https://github.com/dandavison/delta/pull/1615

**Comment by Julien-cpsn** [comment]
Will try to work on that

**Reply by Julien-cpsn** [reply]
My first thoughts are on creating a theme file (TOML) that will contains things like :

```toml
[UI]
main_foreground_color
secondary_foreground_color
main_background_color
secondary_background_color

[Others]
selection_highlight_color
environment_variable_highlight_color

[Methods]
GET
POST
PUT
DELETE
PATCH
OPTIONS
```

**Comment by Julien-cpsn** [comment, marked as answer]
Solved by https://github.com/Julien-cpsn/ATAC/releases/tag/v0.18.0

**Reply by azzamsa** [reply]
Thank you! ❤️
